source/renderer/app/components/widgets/FormattedHTMLMessageWithLink.tsx

```diff
@@ -36,7 +34,7 @@ export class FormattedHTMLMessageWithLink extends Component<Props> {
       </Fragment>
     );
     return linkPosition === 'before'
-      ? [Link, MainMessage]
-      : [MainMessage, Link];
+      ? [Link, <>&nbsp;</>, MainMessage]
+      : [MainMessage, <>&nbsp;</>, Link];
```

Comment:
This change was necessary because otherwise (look above) this component would always add a non-breaking space before _and_ after the main message. I checked in other places of the app and it seems to still work correctly.